src/content/support/overview/index.md

```diff
@@ -0,0 +1,77 @@
+---
+linkTitle: Overview
+title: Customer support
+description: The support we provide is an essential part of our offering. Here we explain various support service processes and workflows.
+weight: 10
+last_review_date: 2022-12-07
+user_questions:
+  - What should I know when working with Giant Swarm's support staff?
+  - How is Giant Swarm organizing support?
+menu:
+  main:
+    parent: support-training
+    weight: 1
+aliases:
+  - /general/support
+  - /basics/giant-swarm-support/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+---
+
+In Giant Swarm we have developed a custom support model. In order to offer a great support we define different layers of assistance.
+
+## Direct Support via Slack
+
+Our first level support relies on close interactions via Slack, to ensure bi-directional feedback as quickly as possible. This interaction channel is also used to answer a variety of questions. The conversations are not limited to the platform itself, but can go broader and include the anything cloud-native.
+
+Our first level support shifts run from 08:00 to 18:00 (CET) on Monday-Friday. While production support goes without saying, we  often answer questions well outside those times. We rotate support shifts across our teams, which is why we focus on support channels with clear internal handovers.
```

Comment:
> While production support goes without saying, we  often answer questions well outside those times

What do you mean by "While production support goes without saying"? That we do oncall? And should we really promise that we answer outside the times? Or do you rather want to say that we are in a close relationship with the customers and because of that know what is important when and also support our customers outside of those hours. 

Do we have business hours that we do for US customers? Shall we add those times here too?

Reply:
> While production support goes without saying

This was on the support page already, written by Oli. Let me rewrite it but it wants to emphasize production is 24/7 meanwhile first level is best effort after business hours